subsys/net/ip/net_if.c

```diff
@@ -2649,6 +2649,10 @@ uint8_t net_if_ipv6_get_mcast_hop_limit(struct net_if *iface)
 
```

Comment:
could this patch go already in? As for the ipv4 one

Reply:
Yes, I will separate the generic fixes after 3.6 is released.

Reply:
> I will separate the generic fixes after 3.6 is released.

I sent the extra commits already to be included in 3.7